config.default.php

```diff
@@ -36,7 +36,7 @@
 	# Example: '<img class="logo" src="https://example.net/Hello.png" alt="Logo Example" /> Hello'
 	'logo_html' => '',
 
-	# Name of the user that has administration rights.
+	# Name of the public user important for anonymous reading..
```

Comment:
```suggestion
	# Name of the default user. Also used as the public user for anonymous reading.
```

Reply:
perhaps the term "anonymous" would be better? The variable is also called "allow_anonymous".
`# Name of the anonymous user. Used as the public user for reading.`

or is the public user still used somewhere else?

Reply:
The default user is used for several things: it cannot be deleted, it is the default for showing e.g. errors when not logged in, etc. One of the features is that this is the one used for anonymous reading